### ISSUE TITLE
Fetch translated categories and tags

### ISSUE THREAD

**Issue by juanramoncarceles** [rburgst/wp-graphql-wpml#4]
Hi! Thanks for your plugin, it has been already very useful for me!
I would like to know if it could also give the translated categories and tags.

Currently when fetching all the posts there are only categories and tags for the posts in the main language. There rest appear empty:

![image](https://user-images.githubusercontent.com/31431405/96724469-27edcd80-13b0-11eb-84f9-cf28d1b65d91.png)

Even if I query specifically for all the categories and tags like below, which would be an alternative, I only get the ones in the main language. It would be great to get all with their locale to be able to filter them.

```graphql
query MyQuery {
  tags {
    nodes {
      name
    }
  }
  categories {
    nodes {
      name
    }
  }
}
```

Maybe I am missing something and this is already possible. They do appear in the translated versions of the posts.

I need to eventually consume this data from Gatsby.

**Comment by rburgst** [comment]
Hi, currently I dont have the bandwidth to look at this. However I am always looking for contributors.

**Comment by rburgst** [comment]
this is hopefully fixed now via #10
